SmartLink.Host/resource_isolation/main.py

```
@@ -0,0 +1,17 @@
+import uvicorn
+import logging
+from src.api.routes import app
```

Ruff (0.8.2)

`src.api.routes.app` imported but unused

Remove unused import: `src.api.routes.app`

(F401)